src/common/config_opts.h

```diff
@@ -332,6 +332,11 @@ OPTION(mon_warn_on_crush_straw_calc_version_zero, OPT_BOOL, true) // warn if cru
 OPTION(mon_warn_on_osd_down_out_interval_zero, OPT_BOOL, true) // warn if 'mon_osd_down_out_interval == 0'
 OPTION(mon_warn_on_cache_pools_without_hit_sets, OPT_BOOL, true)
 OPTION(mon_warn_osd_usage_min_max_delta, OPT_FLOAT, .40) // warn if difference between min and max OSD utilizations exceeds specified amount
+// note that for ec, the minimal sensible configuration is 2 + 1,
+// so 3 is applicable for both replicated pools and ec pools
+OPTION(mon_suggested_pool_size, OPT_U32, 3)
```

Comment:
It's convenient that 3 is what we (i.e., developers) want for both ec and replicated, but I suspect that that may be vary in other cases. Perhaps we want a mon_suggested_replicas and mon_suggested_m (as in k+m) or something?